docs/root/intro/version_history.rst

```diff
@@ -1,22 +1,24 @@
 Version history
 ---------------
 
-1.7.0
-=====
+1.7.0 (Pending)
+===============
 
 * Added :ref:`weighted round robin
```

Comment:
What about this...it seems like "Added" is not a category; should this go into something like "load balancing" and put above "logging"?

Reply:
This is a "bug." I already asked @htuch to fix this. Feel free to fix.